meta-resin-common/recipes-support/resin-device-uuid/resin-device-uuid.bb

```diff
@@ -20,7 +20,15 @@ RDEPENDS_${PN} = " \
 
 SYSTEMD_SERVICE_${PN} = "resin-device-uuid.service"
 
+FILES_${PN} += "/home/root/.rnd"
+
 do_install() {
+    # Create an initial file where openssl will save its state
+    # We will bind mount here a location in resin-conf partition to make it rw
+    mkdir -p ${D}/home/root/
```

Comment:
Should we just add all of the root's home in `resin-conf`? It sounds like something useful for debugging purposes too.